### PR TITLE
Correct jprod and jtprod for SlackModel

### PR DESCRIPTION
**Pull request by abelsiqueira**
Relates to JuliaSmoothOptimizers/NLPModels.jl/issues/32

### CODE REVIEW

**Comment by abelsiqueira** [comment]
Something very odd happened on one of the Travis tests, but the others are passing.

File: .travis.yml [hunk 1]
```diff
@@ -21,9 +21,10 @@ script:
   - which cmake
   - cmake --version
   - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
+  - julia -e 'Pkg.clone(pwd())'
   - julia -e 'Pkg.clone("https://github.com/JuliaSmoothOptimizers/NLPModels.jl.git"); Pkg.checkout("NLPModels", "develop"); Pkg.build("NLPModels")'
   - julia -e 'Pkg.clone("https://github.com/mlubin/AMPLMathProgInterface.jl.git")'
   - julia -e 'Pkg.add("Ipopt")'
-  - julia --check-bounds=yes -e 'Pkg.clone(pwd()); Pkg.build("AmplNLReader"); Pkg.test("AmplNLReader"; coverage=true)'
+  - julia --check-bounds=yes -e 'Pkg.build("AmplNLReader"); Pkg.test("AmplNLReader"; coverage=true)'
```
Comment (dpo) [inline .travis.yml]: What was the reason for this change?

Comment (dpo) [inline .travis.yml]: Oh, nevermind. I just saw line 24.

**Comment by dpo** [comment]
Thank you!

**Pull request merged**